src/features/vote/ActiveRequestsForm.tsx

```diff
@@ -20,6 +20,7 @@ import { formatVoteDataToCommit } from "./helpers/formatVoteDataToCommit";
 import { EncryptedVote } from "web3/get/queryEncryptedVotesEvents";
 
 import { FormWrapper, ModalWrapper } from "./styled/ActiveRequestsForm.styled";
+import { VoteRevealed } from "web3/get/queryVotesRevealedEvents";
```

Comment:
nit: are any of the changes in this file related to the retrieve rewards feature? This isn't a show-stopper on this PR, but in the interest of reducing PR sizes, maybe changes like this should be put into a separate PR in the future?

Thoughts? Or am I misunderstanding the purpose of the changes in this file?

Reply:
Oh this was largely finishing off some work that was tagged onto my larger previous PR. as it was a rather small addition I didn't feel the need to do a PR separate

Highly supportive of smaller PRs, though, we just need a fast turn around.

Reply:
Yeah, I would argue that this is the exact sort of change that warrants its own PR. I think smaller PRs often _help_ with fast turnaround bc reviewers tend to procrastinate on PRs that are 100+ lines in my experience.